react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -172,8 +172,20 @@ const AutocompleteInput = ({
   })
 
   return (
+    <form action="#" onSubmit={e => {
```

Comment:
Do we need this form handler? Or is the input search that is fixing the issue?

Reply:
Absolutely! We need the form tag with action attribute + the input search too to have the search button on the iOs virtual keyboard.